roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -28,11 +28,10 @@ cloud:
 discovery:
   type: kubernetes
   zen.ping.multicast.enabled: false
-  zen.minimum_master_nodes: {{es_min_masters}}
+  zen.minimum_master_nodes: {{es_node_quorum}}
```

Comment:
I will update to use ENV var for consistency and we can change later if that's the path we choose.